chain/rust/src/builders/utils.rs

```diff
@@ -0,0 +1,126 @@
+//use crate::auxdata::{AuxiliaryData, GeneralTransactionMetadata};
+
+/// this value is large enough to trigger cbor_event to force writing a u64 cbor type
+/// when used inside `write_type`.
+/// This is useful to calculate payloads assuming the max possible size that can be used
+/// since u64 in CBOR are fixed size (will prefix with 0s to stay a constant width regardless of value)
+pub const force_u64: u64 = 0x01_00_00_00_00; 
+
+// impl GeneralTransactionMetadata {
```

Comment:
Why is all of this commented out?

Reply:
Not referred to so far. I'll probably remove it after finishing migrating `tx_builder.rs` if it's not needed. That is unless we still want them. It just seemed like more of the helper functionality to help with WASM making many temporaries like the ones we talked about in the other builders.